### ISSUE TITLE
Node.js v16 not working

### ISSUE THREAD

**Issue by juancarlovieri** [SynoCommunity/spksrc#5065]
### Is this a new Bug?

- [x] I checkd that the bug hasn't been reported before

### Package Name

Node.js® v16

### Package Version

Version 16.13.1-1

### Device Model

DSM216play

### Device Architecture

ARMv7

### Firmware Version

DSM 7.0.1-42218

### What happened?

I recently installed the new node.js v16 package. However, every time I run it, it shows this error:
`node: error while loading shared libraries: libatomic.so.1: cannot open shared object file: No such file or directory`
As apt-get is not available in synology, I am not able to install lilibatomic1.
Is anyone getting the same error or is it just me? Thanks in advanced.

### Reproduction steps

```bash
1. install node v16
2. ssh to synology
3. run node
...
```

### Install Log

```log
2022/01/08 10:04:30     install nodejs_v16 16.13.1-1 Begin preinst
2022/01/08 10:04:31     Begin reload_inst_variables
2022/01/08 10:04:31     End reload_inst_variables
2022/01/08 10:04:31     Begin initialize_variables
2022/01/08 10:04:31     End initialize_variables
2022/01/08 10:04:31     ===> Step preinst. USER= GROUP= SHARE_PATH=
2022/01/08 10:04:31     install nodejs_v16 16.13.1-1 End preinst ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/mkdir -p /volume2/@appstore/nodejs_v16
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/mkdir -p /volume2/@appstore/nodejs_v16 ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/rm -rf /volume2/@appstore/nodejs_v16
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/rm -rf /volume2/@appstore/nodejs_v16 ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/package /volume2/@appstore/nodejs_v16
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/package /volume2/@appstore/nodejs_v16 ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/rm -rf /var/packages/nodejs_v16
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/rm -rf /var/packages/nodejs_v16 ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/mkdir -p /var/packages/nodejs_v16
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/mkdir -p /var/packages/nodejs_v16 ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/touch /var/packages/nodejs_v16/installing
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/touch /var/packages/nodejs_v16/installing ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/INFO /var/packages/nodejs_v16/INFO
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/INFO /var/packages/nodejs_v16/INFO ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/rm -rf /var/packages/nodejs_v16/scripts
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/rm -rf /var/packages/nodejs_v16/scripts ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/scripts /var/packages/nodejs_v16/scripts
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/scripts /var/packages/nodejs_v16/scripts ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/rm -rf /var/packages/nodejs_v16/WIZARD_UIFILES
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 End /bin/rm -rf /var/packages/nodejs_v16/WIZARD_UIFILES ret=[0]
2022/01/08 10:04:40     install nodejs_v16 16.13.1-1 Begin /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/WIZARD_UIFILES /var/packages/nodejs_v16/WIZARD_UIFILES
2022/01/08 10:04:41     install nodejs_v16 16.13.1-1 End /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/WIZARD_UIFILES /var/packages/nodejs_v16/WIZARD_UIFILES ret=[0]
2022/01/08 10:04:41     install nodejs_v16 16.13.1-1 Begin /bin/rm -rf /var/packages/nodejs_v16/conf
2022/01/08 10:04:41     install nodejs_v16 16.13.1-1 End /bin/rm -rf /var/packages/nodejs_v16/conf ret=[0]
2022/01/08 10:04:41     install nodejs_v16 16.13.1-1 Begin /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/conf /var/packages/nodejs_v16/conf
2022/01/08 10:04:41     install nodejs_v16 16.13.1-1 End /bin/mv -f /volume2/@tmp/pkginstall/extract.Nzm9e6/conf /var/packages/nodejs_v16/conf ret=[0]
2022/01/08 10:05:13     install nodejs_v16 16.13.1-1 Begin postinst
2022/01/08 10:05:13     Begin reload_inst_variables
2022/01/08 10:05:13     End reload_inst_variables
2022/01/08 10:05:13     Begin initialize_variables
2022/01/08 10:05:14     End initialize_variables
2022/01/08 10:05:14     ===> Step postinst. USER= GROUP= SHARE_PATH=
2022/01/08 10:05:14     Begin save_wizard_variables
2022/01/08 10:05:14     End save_wizard_variables
2022/01/08 10:05:14     Begin syno_sync_var_folder
2022/01/08 10:05:14     End syno_sync_var_folder
2022/01/08 10:05:14     install nodejs_v16 16.13.1-1 End postinst ret=[0]
2022/01/08 10:05:23     install nodejs_v16 16.13.1-1 Begin start-stop-status start
2022/01/08 10:05:23     install nodejs_v16 16.13.1-1 End start-stop-status start ret=[0]
```

**Comment by hgy59** [comment]
unfortunately I do not have an ARMv7 device with DSM 7.0.
On DSM 6 there is only `/usr/lib/libatomic_ops.so.1` on arm devices, so I suppose we have to build and include libatomic into to the package for ARMv7 archs.

**Comment by Mike1074** [comment]
DS215J also produces this error. 

DSM 7.0.1-42218

Architecture
armv7l

**Comment by hgy59** [comment]
> DS215J also produces this error.

This bug is related to all armv7 archs.

**Comment by hgy59** [comment]
After the github builds succeeds you can download update packages under #5069 - checks - artifacts - packages.zip.
Don't worry that the revision (-1) is not incremented yet. This is no problem with manual package installation.

**Comment by juancarlovieri** [comment]
Okay, thanks for the quick response

**Comment by junheepark0310** [comment]
> After the github builds succeeds you can download update packages under #5069 - checks - artifacts - packages.zip. Don't worry that the revision (-1) is not incremented yet. This is no problem with manual package installation.

Hello! I also have the same issue but I can't find update packages under #5069 - checks - artifacts - packages.zip.

**Comment by hgy59** [comment]
> Hello! I also have the same issue but I can't find update packages under #5069 - checks - artifacts - packages.zip.

github artifacts are deleted 90 days after creation.

You should take the latest version published as v16.17.0-4 

To install our nodejs v16 package in the package center you need to configure the synocommunity package repository as documented on https://synocommunity.com/.

_(or you can manually download the package under https://synocommunity.com/package/nodejs_v16)_
